apps/idos-enclave/src/lib/enclave.js

```diff
@@ -29,22 +33,30 @@ export class Enclave {
     this.store.reset();
   }
 
-  storage(humanId, signerAddress, signerPublicKey) {
+  async storage(humanId, signerAddress, signerPublicKey) {
+    const permission = await navigator.permissions.query({
+      name: "storage-access",
+    });
```

Comment:
How well have you seen this being supported?

My first instinct here wouldn't be to try to reach in to unpartitioned state from the iframe. Instead, I'd show and communicate with the pop-up. Happy to leave this as-is if you find it less confusing.